sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/cosmos/CosmosHealthIndicator.java

```diff
@@ -44,19 +45,33 @@ public CosmosHealthIndicator(CosmosAsyncClient cosmosAsyncClient, String databas
     }
 
     @Override
-    protected void doHealthCheck(Builder builder) {
-        CosmosDatabaseResponse response = this.cosmosAsyncClient.getDatabase(database)
-            .read()
-            .block(timeout);
-
-        if (response != null) {
-            LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
-                response.getRequestCharge(), endpoint, database);
-        }
-        if (response == null) {
-            builder.down();
+    protected void doHealthCheck(Builder builder) throws Exception {
+        if (database == null) {
+            builder.status("The option of `spring.cloud.azure.cosmos.database` is not configured!");
```

Comment:
Can we use the `Status.UNKNOWN` and put this information in the builder.withDetail, just like how https://github.com/Azure/azure-sdk-for-java/blob/5bc550c9a5de4f8ee93a5b3141500ee34be3850d/sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/eventhubs/EventHubsHealthIndicator.java#L39 does.

Reply:
yes